google/provider_test.go

```diff
@@ -53,6 +53,7 @@ func testAccPreCheck(t *testing.T) {
 		"GOOGLE_CREDENTIALS",
 		"GOOGLE_CLOUD_KEYFILE_JSON",
 		"GCLOUD_KEYFILE_JSON",
+		"GOOGLE_USE_DEFAULT_CREDENTIALS",
```

Comment:
Does this also need to be added to `provider.go` so that the credentials will be picked up? Unless I'm mistaken this section here just pre-validates that we have some kind of credentials to move forward with. Are they picked up somewhere else that I'm not seeing?